### DISCUSSION TITLE
[Unstable] Nerf regeneration of Combat stimulant

### DISCUSSION THREAD

**Discussion by N1ght-Raven** [FakeFishGames/Barotrauma#7029] [Unstable feedback]
Combat stimulant gives crazy regeneration and can give you nearly invincibility in fights. You literally can survive several bites from Endworm or Charybdis and return to the sub with full HP. I think this thing would regenerate wounds much slower and not healing the bloodloss. But to compensate that nerf, that stimulant will give you more meele power, meele attack speed and boost your fire rate.
Also I think this stim a bit too simple to craft. Espetially for medic with xenology talants.

**Comment by LePiromano** [comment]
have you tried deusizine?

**Reply by N1ght-Raven** [reply]
> have you tried deusizine?

Yes. Deisizine heals much less effective and not heals bleeding and burns unlike Combat stimulant
